events/wc_events/wc_lordaeron_events.txt

```diff
@@ -0,0 +1,410 @@
+
+		show_as_tooltip = { 
+			custom_tooltip = lordaeron.1001.tooltip.a
+		}
+
+		hidden_effect = { 
+			global_var:sylvanas_character = { 
+				trigger_event = { # replace with on-action for redirect?? or just use trigger_fail???
+					id = lordaeron.1101 #selenite texts found in undercity; revival imminent
+					years = 2
+					delayed = yes
+				}
+			}
+			while = { 
+				count = 3 # in three different courts...
+				random_vassal = { 
+					limit = { 
+						faith.religion = { is_in_family = rf_shadow }
+						NOT = { government_has_flag = government_is_theocracy }
+					}
+					while = {
+						count = 4 # four characters will adopt Forgotten Shadow as secret faith
+						random_pool_character = { 
+							province = this.capital_province
+							limit = {
+								NAND = { 
+									has_council_position = councillor_court_chaplain
+									NOT = { faith = faith:forgotten_shadow }
+									NOT = { government_has_flag = government_is_theocracy }
+									NOT = { piety_level >= 2 }
+									NOT = { exists = secret_faith }
+									NAND = { has_trait = zealous has_trait = loyal has_trait = holy_warrior has_trait = cynical }
+								}
+							}
+							make_character_crypto_religionist_effect = { CRYPTO_RELIGION = faith:forgotten_shadow }
+						}
+					}
+				}
+			} #total: 12 characters will adopt Forgotten Shadow secretly
+			#TODO: Link the random vassals to the next event? could be cool to have secret faith characters randomly dispersed tho
+		}
+	}
+
+}
+
+# Failed construction of Undercity
+lordaeron.1002 = { 
+
+	type = character_event
+	title = lordaeron.1002.title
+	desc = lordaeron.1002.desc
+	theme = dungeon
+	override_background = market
+
+	right_portrait = { 
+		character = root
+		animation = stress
+	}
+
+	immediate = { 
+		title:b_lordaeron.title_province = {
+			remove_province_modifier = wc_building_undercity
+		}
+		hidden_effect = { 
+			trigger_event = {
+				on_action = undercity_reset_construction
+				months = {29 31}
+			}
+		}
+	}
+
+	option = { 
+		name = lordaeron.1002.opt.a
+		show_as_tooltip = { 
+			custom_tooltip = lordaeron.1002.tooltip.a
+		}
+	}
+	
+}
+
+#####################################
+#1100 - 1199: Post-Reclamation events
+#####################################
+
+#1100 - 1109: Selenite Revival
+
+#lordaeron.1100 = { } Selenite Revival Redirect event in case Lordaeron changes hands
+
+lordaeron.1101 = { 
+
+	type = character_event
+	title = lordaeron.1101.title
+	desc = lordaeron.1101.desc
+	theme = faith
+	override_background = wc_undercity_throne_room
+
+	immediate = {
+		every_vassal_or_below = {
+			limit = {
+				AND = { 
+					faith.religion = { is_in_family = rf_shadow }
+					highest_held_title_tier >= tier_county
+					NOT = { faith = faith:forgotten_shadow }
+					NOT = { government_has_flag = government_is_theocracy }
+					NOT = { piety_level >= 2 }
+					NAND = { has_trait = zealous has_trait = loyal has_trait = holy_warrior has_trait = cynical }
+				}
```

Comment:
```suggestion
				faith.religion = { is_in_family = rf_shadow }
				highest_held_title_tier >= tier_county
				NOT = { faith = faith:forgotten_shadow }
				NOT = { government_has_flag = government_is_theocracy }
				NOT = { piety_level >= 2 }
				NOR = { has_trait = zealous has_trait = loyal has_trait = holy_warrior }
```
you meant this probably?

Reply:
yes and... should i be using NOR for these quadruple NOTs ?

Reply:
yes, will be better